hri_safety_sense/src/VscProcess.cpp

```diff
@@ -36,186 +35,215 @@
 #include "hri_c_driver/VehicleInterface.h"
 #include "hri_c_driver/VehicleMessages.h"
 
+using namespace std::placeholders;
 using namespace hri_safety_sense;
```

Comment:
Get rid of both of these `using namespace`, please.

Reply:
7de7162